commands/utility/quote.js

```diff
@@ -72,12 +73,12 @@ exports.run = async (handler, message, args, pre) => {
 		}
 	}
 
-	if (quoteText.length > DESCRIPTION_LIMIT) {
-		quoteText = quoteText.slice(0, DESCRIPTION_LIMIT - 1) + '…'
+	if (quoteText.length > DESCRIPTION_LIMIT - quotePermalink.length) {
```

Comment:
Could quotePermalink.length be greater than DESCRIPTION_LIMIT? We might want some boundary checks to make sure we are not slicing from the back with the following .slice()